File: src/main/java/net/arvandor/numinoustreasury/item/action/NuminousOnInteractAir.java
package net.arvandor.numinoustreasury.item.action;

import org.bukkit.configuration.serialization.ConfigurationSerializable;
import org.bukkit.event.player.PlayerInteractEvent;

public interface NuminousOnInteractAir extends ConfigurationSerializable {
    void onInteractAir(PlayerInteractEvent event);
}

File: src/main/kotlin/net/arvandor/numinoustreasury/item/action/NuminousOnInteractAir.kt
package net.arvandor.numinoustreasury.item.action

import org.bukkit.configuration.serialization.ConfigurationSerializable
import org.bukkit.event.player.PlayerInteractEvent

interface NuminousOnInteractAir : ConfigurationSerializable {
    fun onInteractAir(event: PlayerInteractEvent)
}
